src/components/user/uc/users.uc.js

```diff
@@ -1,6 +1,8 @@
 const { ObjectId } = require('mongoose').Types;
 const { BadRequest, Forbidden } = require('../../../errors');
-const { userRepo, accountRepo, trashbinRepo } = require('../repo/index');
+const { userRepo, accountRepo, trashbinRepo } = require('../repo');
+const registrationPinUC = require('./registrationPin.uc');
+const pseudonymUC = require('../../pseudonym/uc/pseudonym.uc');
```

Comment:
either we should implement and use a facade for the pseudonym UC, or move the pseudonym uc into the users component.
Importing a UC from a different component wont do.